testUtils/api.ts

```diff
@@ -1,17 +1,28 @@
-import { InMemoryCache } from "apollo-cache-inmemory";
 import { BatchHttpLink } from "apollo-link-batch-http";
-import ApolloClient from "apollo-client";
+// import ApolloClient from "apollo-client";
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why?
It's not extraneous.

Reply:
Because I'm getting `'setup-polly-jest' should be listed in the project's dependencies, not devDependencies.`